crates/pixi_api/src/context.rs

```diff
@@ -16,6 +20,10 @@ impl<I: Interface> WorkspaceContext<I> {
         }
     }
 
+    pub fn workspace(&self) -> Workspace {
+        self.workspace.clone()
+    }
```

Comment:
Wouldn't it make more sense to return `&Workspace` and the caller has to `clone` themselves if necessary?